content/en/core/releases/4.11.0.md

```diff
@@ -29,7 +29,7 @@ None.
 The CHT datasource API has been expanded to allow querying places and persons by contact type.
 
 ##### Details
-See more details in the [api docs](http://docs.communityhealthtoolkit.org/cht-datasource/).
+See more details in the [API docs](https://docs.communityhealthtoolkit.org/building/reference/api/#get-apiv1person).
```

Comment:
```suggestion
See more details in the [API docs]({{< relref "building/reference/api#get-apiv1person" >}}).
```

Reply:
ugh - that was lazy of me - thanks for the fix!